src/client/src/components/pages/dashboard/index.jsx

```diff
@@ -23,6 +23,10 @@ export default class index extends Component {
       console.log(err); // waiting for boundery error handling
```

Comment:
why you need this `console.log(err); // waiting for boundery error handling` ?!

Reply:
I think you reviewed Asala branch
She made this handling error :smile: